app/(main)/_components/track.tsx

```diff
@@ -4,13 +4,27 @@ import "./track.css";
 
 interface ITrackProps {
   name: string;
-  imageSrc: string;
 }
 
-const Track: React.FC<ITrackProps> = ({ name, imageSrc }) => {
+const Track: React.FC<ITrackProps> = ({ name }) => {
   return (
     <div className="flex flex-col items-center space-y-4 md:hover:scale-105 transition-transform duration-300 md:cursor-pointer">
```

Comment:
```suggestion
    <div className="flex flex-col items-center space-y-4 md:hover:scale-105 transition-transform duration-300">
```

Looks good! This is the only other change I'd suggest implementing. Even if it has a hover effect, if something doesn't have an action or link then it shouldn't have a mouse pointer to avoid confusion for the user.